applications/StructuralMechanicsApplication/custom_conditions/surface_load_condition_3d.cpp

```diff
@@ -134,7 +134,7 @@ namespace Kratos
 
     void SurfaceLoadCondition3D::MakeCrossMatrix(
```

Comment:
just a comment:: shouldn't this function be in the math utils? (no need to change it right now)